internal/gcsx/bucket_manager.go

```diff
@@ -175,13 +164,17 @@ func (bm *bucketManager) SetUpBucket(
 	if name == canned.FakeBucketName {
 		b = canned.MakeFakeBucket(ctx)
 	} else {
-		b, err = bm.SetUpGcsBucket(name)
-		if err != nil {
-			err = fmt.Errorf("OpenBucket: %w", err)
-			return
-		}
+		b = bm.storageHandle.BucketHandle(name, bm.config.BillingProject)
 	}
 
+	// Enable monitoring.
+	if bm.config.EnableMonitoring {
```

Comment:
any reason for keeping this above "b = storage.NewDebugBucket(b)" ?

Reply:
Yeah, but not any strong reason.
(a) Keeping the gcs debug logs and stackdriver metrics order same as fuse logs.
(b) Logically, to avoid the logging time in stackdriver metrics (specially latency related metrics).